websites/L/LCPDFR/presence.ts

```diff
@@ -0,0 +1,95 @@
+const presence = new Presence({
+		clientId: "1133734950055714866",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let profilePictureLink: HTMLAnchorElement,
+	urlParams: URLSearchParams,
+	searchQuery: string;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/WDOURvX.png",
+			startTimestamp: browsingTimestamp,
+		},
+		privacy = await presence.getSetting<boolean>("privacy");
```

Comment:
Please destructure it as it is used a lot in the code.
```suggestion
		privacy = await presence.getSetting<boolean>("privacy"),
		{search, pathname, hostname} = document.location;
```